source/module_basis/module_nao/radial_set.h

```diff
@@ -104,6 +104,7 @@ class RadialSet
                        const int,
                        const double,
                        const double = 0.1,
+                       const double = 0.01,
```

Comment:
Maybe parameter lists should have names or at least comments.

Reply:
Good advice! Comments have been added.